d2bs/kolbot/libs/bots/BattleOrders.js

```diff
@@ -1,12 +1,63 @@
 /**
 *	@filename	BattleOrders.js
-*	@author		kolton
+*	@author		kolton, jmichelsen
 *	@desc		give or receive Battle Orders buff
+*	@return {boolean}
```

Comment:
can you remove this? =) If so we are good to go if you ask me

Reply:
Thanks J :) 

Is there a negative to having this return type specified? I added it to satisfy the linter that complains that "Primitive value returned from constructor will be lost when called with 'new' "

Reply:
You got the right linter file?🙃

Reply:
Sorry, it's actually Webstorm that complains when you don't specify a return in the docstring on functions like this. Still, will it cause problems to leave it in? I can pull it out either way, if that's what we'd prefer. Just curious.

Reply:
I removed this. It's ready for rereview or approval :)